app/errors.py

```
@@ -0,0 +1,18 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    def __init__(self, message="Non vaccinated! You can`t go to cafe") -> None:
+        super().__init__(message)
+    pass
+
+
+class NotWearingMaskError(Exception):
+    def __init__(self, message="Where is your mask? you have to wear the mask") -> None:
```

The default message 'Where is your mask? you have to wear the mask' should start with a capital letter for 'you'. Consider changing it to 'Where is your mask? You have to wear a mask.'